pallets/ah-migrator/src/lib.rs

```diff
@@ -971,6 +984,18 @@ pub mod pallet {
 			Self::do_receive_recovery_messages(messages).map_err(Into::into)
 		}
 
+		#[cfg(feature = "kusama-ahm")]
+		#[pallet::call_index(27)]
+		#[pallet::weight(T::AhWeightInfo::receive_staking_messages(messages.len() as u32))] // TODO @muharem weight
```

Comment:
It was a bit difficult to get it working with a feature-gate for the recovery pallet, maybe we just estimate it here.